amplipi/rt.py

```diff
@@ -373,24 +387,35 @@ def read_temps(self, preamp: int = 1) -> Tuple[Union[float, None], Union[float,
       temp_hv1 = self._fix2temp(temp_hv1_f)
       temp_amp1 = self._fix2temp(temp_amp1_f)
       temp_amp2 = self._fix2temp(temp_amp2_f)
-      return temp_hv1, temp_amp1, temp_amp2
-    return None, None, None
+      if self.hv2_present(preamp):
+        temp_hv2_f = self.bus.read_byte_data(preamp*8, _REG_ADDRS['HV2_TEMP'])
+        temp_hv2 = self._fix2temp(temp_hv2_f)
+        return temp_hv1, temp_hv2, temp_amp1, temp_amp2
+      else:
+        return temp_hv1, None, temp_amp1, temp_amp2
+    return None, None, None, None
 
-  def read_hv(self, preamp: int = 1) -> Union[float, None]:
+  def read_hv(self, preamp: int = 1) -> Union[Tuple[float, float], Tuple[float, None], Tuple[None, None]]:
```

Comment:
```suggestion
  def read_hv(self, preamp: int = 1) -> Tuple[Optional[float], Optional[float]]:
```
Although this signature isn't as specific it is specific enough and much easer to read